integration/src/test/java/integration/JobRoleStepDefinitions.java

```diff
@@ -22,13 +24,20 @@ public void employeeViewsJobRolesPage() {
     @When("I select a role")
     public void selectJobRole() {
         JobRolesPage jobRolesPage = new JobRolesPage(driver);
-        jobRolesPage.clickJobRole();
+        jobRolesPage.clickJobRole2();
     }
 
     @Then("I can view the role page")
     public void viewJobRole() {
         JobRolePage jobRolePage = new JobRolePage(driver);
-        assertEquals(jobRolePage.getHeading(), "Product Principal");
+        assertEquals(jobRolePage.getHeading(), "Solutions Architect");
         assertEquals(jobRolePage.getBandName(), "Apprentice");
     }
+
+    @And("I can click on the sharepoint page")
```

Comment:
We don't normally use the `@And` annotation. Stick with `@Given`, `@When` and `@Then`

(Note that if you have `Given something / And something else` then "something else" is also a Given – "And" just repeats whatever came before but in a more readable way.)